plugins/catalog-backend-module-github/src/providers/GithubEntityProviderConfig.ts

```diff
@@ -85,6 +86,10 @@ function readProviderConfig(
 
   const catalogPathContainsWildcard = catalogPath.includes('*');
 
+  const visibilitiesInclude = config?.getOptionalStringArray(
+    'filters.visibilities',
```

Comment:
Naming wise I feel it might be a bit cleaner to call this `filters.visibility` instead? Whether it can be one or many I feel is implied in the fact that it's a string array filter.

Reply:
sounds good